jitbuilder/release/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright IBM Corp. and others 2017
+# Copyright IBM Corp. and others 2023
```

Comment:
The year should not have been changed; it communicates when the file was first added.

Reply:
yeah, that's my bad, i should have caught that.